src/main/java/org/broadinstitute/dsm/model/elasticsearch/ElasticSearchParticipantDto.java

```diff
@@ -67,6 +68,10 @@
         return Optional.ofNullable(dsm);
     }
 
+    public Optional<List<String>> getGovernedUsers() {
```

Comment:
I think it is better idea to return empty list from getters, if field is type of `List`, like:
```
public List<String> getGovernedUsers() {
    if (this.governedUsers == null) {
        this.governedUsers = Collections.emptyList();
    }
    return this.governedUsers;
}
```
I see that there are other getters which return `List` types wrapped in `Optional`, but I think it will be good idea if we change them as well, since returning empty list instead of wrapping it in `Optional` will be more convenient for use and also it will help us to avoid NPEs as well.